### DISCUSSION TITLE
Change User Without Messing Everything Up?

### DISCUSSION THREAD

**Discussion by vipul128** [tsightler/ring-mqtt#558] [Support Q&A] [answered]
So I got around to installing and setting this up over the weekend and really like the integration.  Makes things so much easier than having to go into a separate app!

Anyway, as I was using it I realized I should have created a separate user in Ring and linked that account instead of linking my own account since all notifications will now say my user set the alarm or disarmed instead of saying it was from Home Assistant.

My question is if there is a way to change the user (Ring account) for the integration without royally screwing everything up with duplicate devices and entities, etc.  In short, is there a quick and easy way to do this or should I just suck it up and keep it how it is now because it's not worth the hassle?

Thanks in advance.

**Comment by tsightler** [comment, marked as answer]
Just open the web UI where you did the initial configuration and click the button for force creation of a new refresh token, that is all that is really needed.  It won't make any duplicate devices or anything, heck, even removing and re-adding the addon won't cause any issues since the addon uses predictable topics and consistent IDs for the device registry.  

One important note is that there are a small number of features that will only work when used with the primary account, Base Station volume, Chimes, and Smart Lighting groups.  Overall pretty minor, but worth being aware of as there's the possibility that you will have a few "ghost" devices that were availabel with the primary account but are not available with a secondary account.  They are easy enough to remove from HA, but that's the only real side-effect.

**Reply by vipul128** [reply]
Awesome, thanks for the quick reply and the info on the couple of gotchas if I end up switching the user!
